plugin/backend_linux.go

```diff
@@ -641,7 +641,10 @@ func (pm *Manager) Remove(name string, config *types.PluginRmConfig) error {
 
 	removeDir := pluginDir + "-removing"
```

Comment:
I suggest not ignoring `IsExist` error, but instead do this:
1 generate a random suffix for `removeDir` (instead of a const like it is now)
2 repeat Rename with a different name in case we got `IsExist` error (up to a few times)
3 return IsExist error if it's persistently repeating  (which should not be the case in practice)